MapboxNavigation/VisualInstruction.swift

```diff
@@ -11,7 +11,7 @@ extension VisualInstruction {
 
 #if canImport(CarPlay)
     @available(iOS 12.0, *)
-    var maneuverImageSet: CPImageSet? {
+    func maneuverImageSet(side: DrivingSide = .right) -> CPImageSet? {
```

Comment:
Is the default argument value needed? Seems like more of a trap than a convenience.